virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc

```diff
@@ -4,9 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-installing-qemu-guest-agent
 toc::[]
 
-The QEMU guest agent is a daemon that runs on the virtual machine. The agent
-passes network information on the virtual machine, notably the IP address of
-additional networks, to the host.
+The xref:../../../virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc#virt-viewing-qemu-guest-agent-web[QEMU guest agent] is a daemon that runs on the virtual machine and passes information to the host about the virtual machine, users, and file systems.
```

Comment:
I just noticed this is under "Virtual machine networking" section. Could we move it out? It is not network specific.

Reply:
Content was moved out to new location.